.github/scripts/README.md

```diff
@@ -0,0 +1,84 @@
+# pytorch/.github
+
+> NOTE: This README contains information for the `.github` directory but cannot be located there because it will overwrite the
+repo README.
+
+This directory contains workflows and scripts to support our CI infrastructure that runs on Github Actions.
+
+## Workflows
+
+- Pull CI (`pull.yml`) are run on PRs and on master.
+- Trunk CI (`trunk.yml`) are run on trunk to validate incoming commits. They are usually more expensive to run so we do not run them on PRs unless specified.
+- Scheduled CI (`periodic.yml`) is a subset of trunk CI that is run every few hours on master.
+- Binary CI is run to package binaries for distribution for all platforms.
+
+## Templates
+
+Templates written in [Jinja](https://jinja.palletsprojects.com/en/3.0.x/) are located in the `.github/templates` directory
+and used to generate workflow files for binary jobs found in the `.github/workflows/` directory. These are also a
+couple of utility templates used to discern common utilities that can be used amongst different templates.
+
+### (Re)Generating workflow files
+
+You will need `jinja2` in order to regenerate the workflow files which can be installed using:
+```bash
+pip install -r .github/requirements.txt
+```
+
+Workflows can be generated / regenerated using the following command:
+```bash
+.github/regenerate.sh
+```
+
+### Adding a new generated workflow
+
+New generated workflows can be added in the `.github/scripts/generate_ci_workflows.py` script. You can reference
+examples from that script in order to add the workflow to the stream that is relevant to what you particularly
+care about.
+
+Different parameters can be used to acheive different goals, i.e. running jobs on a cron, running only on trunk, etc.
```

Comment:
This section is getting phased out (we want to enable people to just modify the workflow files themselves when adding a workflow). Maybe just exclude this section for this PR and add the section of adding a workflow later.

Reply:
Not necessarily since binary builds will still be subject to this